src/main/java/com/github/wovnio/wovnjava/HtmlChecker.java

```diff
@@ -24,6 +24,14 @@ public boolean canTranslateContent(String html) {
         return isHtml(head) && !isAmp(head);
     }
 
+    public boolean isTextContentType(String type) {
+        return type == null
+            || type.toLowerCase().contains("text")
```

Comment:
put `type.toLowerCase()` in a variable to avoid repeating it, even if it makes the code less nice.